### PR TITLE
Summary work.

### PR DESCRIPTION
**Pull request by embaldridge**
Worked on summary, fixed the references problem for chapter 3.

### CODE REVIEW

File: summary.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+The species abundance distribution is one of the oldest and most well studied patterns in ecology [@mcgill2007]. Despite the extensive study of this pattern, it remains an open question as to whether the pattern contains enough information to allow the operation of biological processes to be inferred from its shape [@mcgill2007; @harte2011; @white2012; @locey2013; @connolly2014]. I have compiled the largest species abundance distribution dataset ever evaluated and used it to both compare a suite of different models for the SAD and to evaluate whether a particular process can be detected as dominant in structure for a large set of SADs.
```
Comment (ethanwhite) [inline summary.md]: "dominant in structure for" -> "dominant in structuring"

File: summary.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+In general, I found that it is difficult to infer process from species abundance distributions alone.  Part of the difficulty in identifying pattern generating mechanisms from species abundance distributions is due to the fact that multiple mechanisms have been proposed for each formulation of the species abundance distribution. In other words, it is possible for different processes to yield exactly equivalent models [@pielou1975; @mcgill2007]. Subsuming some of these differences into broad categories such as neutral or non-neutral [sensu @connolly2014] may make it possible to draw inferences on general categories of models. However, my results suggest that it may be difficult in general to distinguish among even these broad categories of models and their associated distributions, at least in terrestrial systems. In combination with the results of Connolly et al. (2014), this suggests that, in terrestrial systems, there may not be one single suite of processes that have equal importance in all communities, i.e., non-neutral processes may be more important in some communities, but not in others.
```
Comment (ethanwhite) [inline summary.md]: "it may be difficult in general" -> "it may be difficult"

**Comment by ethanwhite** [comment]
Looks great. I've made a couple of optional suggestions in-line. Ping me when you're happy with the state of those sentences and I'll get this merged.

**Comment by embaldridge** [comment]
Ping.

Elita Baldridge
Ph.D. candidate
Utah State University
Department of Biology and the Ecology Center

**Pull request merged**